20-minimal/Dockerfile.c8s

```diff
@@ -0,0 +1,72 @@
+FROM ubi8/ubi-minimal:latest
+
+# FIXME:
+ADD https://gitlab.cee.redhat.com/platform-eng-core-services/internal-repos/-/raw/master/centos/c8s/c8s-compose.repo /etc/yum.repos.d/c8s-compose.repo
```

Comment:
CentOS Stream 8 host provided by Testing Farm team, that we have selected, do not have access to internal repositories, like https://gitlab.cee.redhat.com.